test.js

```diff
@@ -79,6 +79,42 @@ it('It should log if file is invalid with custom options', function (cb) {
 	file('valid.styl');
 });
 
+it('It should not log if file is valid with custom inline options', function (cb) {
+	var log = sinon.spy();
+	stream = stylint({
+		config: {
+			"colons": "always",
```

Comment:
No problem! I just added the rule to JSHint, so in the future it'll break the build earlier :laughing: